pom.xml

```diff
@@ -501,6 +501,12 @@
                 <version>${org.wso2.carbon.identity.organization.management.core.version}</version>
                 <scope>provided</scope>
             </dependency>
+            <dependency>
```

Comment:
Change the license header with year range for existing files that you edit.
Check other places as well

Reply:
Done